redisinsight/ui/src/pages/browser/components/string-details/StringDetails.spec.tsx

```diff
@@ -111,6 +134,26 @@ describe('StringDetails', () => {
     expect(textArea).toHaveValue(STRING_VALUE_SPACE)
   })
 
+  it('should render "load all"/"download" btn if long string is partially loaded', () => {
```

Comment:
I recommend adding tests after clicking on load button an download button
1. `onRefresh` and `sendTelemetry` should be called
2. actions after `fetchDownloadStringValue` should be in the store